tests/engines/test_pil.py

```diff
@@ -13,8 +13,8 @@
 from io import BytesIO
 from os.path import abspath, dirname, join
 from unittest import TestCase, mock
-import piexif
 
+import piexif
```

Comment:
Given what @scorphus mentioned, piexif might not be available in all platforms.
We can make it optional so it doesn't fail.

Reply:
Quote from [documentation](https://piexif.readthedocs.io/en/latest/about.html#dependency):  "Piexif doesn’t depend on any third library"

There are packages for it:

- [Debian](https://packages.debian.org/sid/python3-piexif)
- [Fedora](https://src.fedoraproject.org/rpms/python-piexif)

Reply:
Thanks for the extra info, @marcelometal.

So, my take is that, with `piexif` depending on no third party lib and being readily available and already packaged in many distros, I think we're good to go with it being a hard dependency. That outweighs the extra logic needed to workaround it being unavailable. We need to remember this when drafting a new release, with the proper information about the new dependency.

Now, speaking about the feature. I believe that not every user would want it — much the opposite, if I'd have to guess. Perhaps, most users would like to strip **all** exif information, regardlessly, leaving no traces behind, as they don't need to keep any at all.

So, I still think the feature of "keeping copyright information when stripping exif data" should stay behind a configuration entry, with default to false/inactive.

Reply:
Thanks guys. @scorphus Makes sense to revert the feature that was added in https://github.com/thumbor/thumbor/pull/1524 in this PR?

Reply:
That can and should definitely be gone granted we do not draft a new release until the issue it fixed is solved otherwise. 😊